test/org/opensolaris/opengrok/configuration/RuntimeEnvironmentTest.java

```diff
@@ -170,7 +170,7 @@ public void run() {
         });
         t.start();
         t.join();
-        assertEquals(new File(path).getCanonicalFile().getAbsolutePath(), instance.getDataRootPath());
+        assertEquals(new File(path), new File(instance.getDataRootPath()));
```

Comment:
my /tmp folder was symlinked and had errors from canonical path. 
This test is more resilient.